internal/libintl.cpp

```diff
@@ -58,8 +59,19 @@ libintl_lite_bool_t loadMessageCatalog(const char* domain, const char* moFilePat
 
 	FILE* moFile = NULL;
 	CloseFileHandleGuard closeFileHandleGuard(moFile);
-	moFile = fopen(moFilePath, "rb");
-	
+
+	string moFilePathStr = moFilePath;
```

Comment:
Please rename `moFilePath` as something like `moFilesRoot`. If I remember well "path" was used to refer to a full path here.